mollie/api/objects/payment.py

```diff
@@ -152,16 +152,16 @@ def can_be_refunded(self):
     @property
     def get_amount_refunded(self):
         try:
-            return float(self._get_property('amountRefunded'))
-        except TypeError:
-            return 0.0
+            return self['amountRefunded']
+        except KeyError:
+            return '0.0'
 
     @property
     def get_amount_remaining(self):
         try:
-            return float(self._get_property('amountRemaining'))
-        except TypeError:
-            return 0.0
+            return self['amountRemaining']
```

Comment:
Use `self._get_property(...) or 0.0`. Also, why is this a string suddenly?

Reply:
https://docs.mollie.com/reference/v2/payments-api/get-payment
Here is documented that `amountRefunded` is returned as a string